sci-geosciences/qgis/qgis-2.8.1.ebuild

```diff
@@ -119,10 +119,12 @@ src_install() {
 		doins -r "${WORKDIR}"/qgis_sample_data/*
 	fi
 
-	python_fix_shebang "${D}"/usr/share/qgis/grass/scripts
-	python_optimize "${D}"/usr/share/qgis/python/plugins \
-		"${D}"/$(python_get_sitedir)/qgis \
-		"${D}"/usr/share/qgis/grass/scripts
+    if use grass; then
+        python_fix_shebang "${D}"/usr/share/qgis/grass/scripts
+        python_optimize "${D}"/usr/share/qgis/python/plugins \
```

Comment:
this shouldn't be under USE grass conditional, but just "${D}"/usr/share/qgis/grass/scripts :+1: 

         python_optimize "${D}"/usr/share/qgis/python/plugins \
         "${D}"/$(python_get_sitedir)/qgis

should stay outside the if, while "${D}"/usr/share/qgis/grass/scripts can be optimized inside the if